### ISSUE TITLE
Bug: Keyboard navigation doesn't work on the atef config GUI tree

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/atef#38]
## Expected Behavior

Keyboard navigation should work

## Current Behavior

Keyboard navigation does not work

## Context

> Bug:
> Keyboard navigation doesn't work on the tree. Perhaps the wrong signal?

_Originally posted by @klauer in https://github.com/pcdshub/atef/pull/28#discussion_r853303449_

**Comment by tangkong** [comment]
This works currently for me, with the left and right arrows jumping up and down depth levels, and up and down moving to between "sequential" items regardless of depth.

This was probably addressed in Zach's big GUI catch-up PR

**Comment by ZLLentz** [comment]
At some point I swapped from a click-only signal to one that also reponds to keyboard inputs for picking a page to show. I don't fully remember all the details.
